doc/source/ecosystem.rst

```diff
@@ -322,12 +322,14 @@ Cyberpandas provides an extension type for storing arrays of IP Addresses. These
 arrays can be stored inside pandas' Series and DataFrame.
 
 `pint`_
-~~~~~~~~~~~~~~
+~~~~~~~
+
+`Pint <https://pint.readthedocs.io/en/latest/>` provides an extension type for
+storing numeric arrays with units. These arrays can be stored inside pandas'
+Series and DataFrame. Operations between Series and DataFrame columns which
+use pint's extension array are then units aware.
 
-Pint provides an extension type for storing numeric arrays with units. These
-arrays can be stored inside pandas' Series and DataFrame. Operations between
-Series and DataFrame columns which use pint's extension array are then units
-aware.
+Add comment about minimum Pint version required once PR merged (see https://github.com/hgrecco/pint/pull/684)
```

Comment:
This line seems out of place.  Is the idea that this PR goes on hold until the Pint PR is merged?

Reply:
yes sorry I should have changed to WIP earlier. 

@jreback's review asks for a minimum required Pint version. As I don't know what that version will be until the PR goes through, this PR also has to go on hold.